docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,99 @@
+
+- Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
+- macOS
+- Windows
+- 32-bit microcontrollers (limited support)
+
+Examples of computers that can run Viam:
+
+{{< board-carousel >}}
+
+## What hardware does Viam support?
+
+In short, anything.
+
+Viam supports a wide variety of sensors, cameras, and other physical hardware, with APIs for each of the following types of hardware:
+
+{{< cards >}}
+{{% relatedcard link="/dev/reference/apis/components/arm/" %}}
+{{% relatedcard link="/dev/reference/apis/components/base/" %}}
+{{% relatedcard link="/dev/reference/apis/components/board/" %}}
+{{% relatedcard link="/dev/reference/apis/components/button/" %}}
+{{% relatedcard link="/dev/reference/apis/components/camera/" %}}
+{{% relatedcard link="/dev/reference/apis/components/encoder/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gantry/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gripper/" %}}
+{{% relatedcard link="/dev/reference/apis/components/input-controller/" %}}
+{{% relatedcard link="/dev/reference/apis/components/motor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/movement-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/power-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/servo/" %}}
+{{% relatedcard link="/dev/reference/apis/components/switch/" %}}
+{{% relatedcard link="/dev/reference/apis/components/generic/" %}}
+{{< /cards >}}
+
+These standardized APIs are implemented by {{< glossary_tooltip term_id="module" text="modules" >}} that provide drivers for specific models of hardware.
+
+Any hardware that is not already supported by a Viam module can be added into Viam's system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
+
+Viam also supports various software services such as [data capture](/data-ai/capture-data/capture-sync/) and [computer vision](/dev/reference/apis/services/vision/), designed to integrate seamlessly with the hardware driver modules.
+
+## How do I get started?
+
+1. [Set up your machine and connect it to the Viam app](/operate/get-started/setup/).
+
+1. Next, [configure supported hardware](/operate/get-started/supported-hardware/) and software on your machine in the Viam app.
+   You can [share the configuration across multiple machines](/manage/fleet/reuse-configuration/) to scale quickly.
+
+1. To integrate other hardware or software services that aren't already supported, you can [write your own modules](/operate/get-started/other-hardware/).
+
+1. [Test your machine](/operate/get-started/test/) by using the **TEST** panel of each configured component's config card to, for example, view your camera's stream or turn your motor.
+
+1. From there, you have many options including:
```

Comment:
Similarly not sure where to put this, but I think we should remove the "Configure machine settings" and "Reuse device configuration" left nav links. It feels odd to have some next steps here and some in the left nav, and for them to be different. We could optionally fold those links into the next steps section here

Reply:
Okay fair point. Naomi may have thoughts since they're to her section and it won't change the main flow here besides the oddness you duly point out.

Reply:
Sounds good - happy to leave in until Naomi returns

Reply:
@npentrel are you okay with these two being removed from left nav?

Reply:
I worry they won't be discoverable but let's leave them off for now.